### DISCUSSION TITLE
V9: Drawer and Tree components ETA

### DISCUSSION THREAD

**Discussion by blendsdk** [microsoft/fluentui#28292] [Q&A] [answered]
Hi,

Would it be possible to provide an ETA on the Drawer component and the Tree component.
The drawer looks very nice. It just needs some animation.

Thanks.

**Comment by ValentinaKozlova** [comment]
Hello, @blendsdk , could you please provide more details? What animation do you want to have in the Drawer component. And what questions do you have about the Tree component?

**Reply by ValentinaKozlova** [reply]
Hi, @marcosmoura , could you please add your input on this discussion?

**Reply by marcosmoura** [reply, marked as answer]
Hello @blendsdk. The Drawer component will have animations/transitions soon. We are currently testing this behavior and it won't affect the current API. As for the ETA, we don't have an exact date yet. We are working towards making the component stable. I will reply to this thread as soon as I have more updates about it. :)

**Comment by ValentinaKozlova** [comment]
Hi @bsunderhus, could you please add you input on this discussion?

**Reply by bsunderhus** [reply]
Hey @blendsdk, the Tree component is currently under development as unstable and it's API is not quite there yet, so unfortunately we don't have an exact date for the ETA. We're estimating something around late July / Mid August depending on how stable the API will become. The more early adopters the sooner this can be achieved, let me know if you're interested in helping with that 😁

I'll try to keep this thread updated on any changes on the ETA! As for features we're still missing before stable release we only have one: 

1. Selection/multiselection implementation

**Reply by blendsdk** [reply]
@bsunderhus, I am interested in helping, I (sort of) know how the slots API work.

**Reply by bsunderhus** [reply]
That's great to hear @blendsdk! The best way right now for you to help would be by becoming an early adopter! That way you can cooperate with opening issues/pr's with bugfix/feature requests based on what you've used so far, but be aware as it's still unstable a lot of internal changes will happen!

**Comment by anty30** [comment]
Hi,

Just one question about Drawer component. Will there be soon a migration guideline from v8 Panel to Drawer ? 

Thanks in advance for your response

**Reply by marcosmoura** [reply]
The components are fairly different. But we will prioritize creating this guide. I cannot give an ETA now, but we'll come back as soon as we have something.
